tl/features/cell_context_matches.py

```diff
@@ -185,10 +189,13 @@ def initialize(self, input_df, context_dict, label_column):
                                                                      input_df['kg_aliases']):
             kg_id_context = context_dict.get(kg_id, None)
             kg_labels = []
-            if kg_id_label_str and kg_id_label_str.strip() != "":
-                kg_labels.append(kg_id_label_str.strip())
-            if kg_id_alias_str and kg_id_alias_str.strip() != "":
-                kg_labels.append(kg_id_alias_str.strip())
+
+            if not (isinstance(kg_id_label_str, float)):
```

Comment:
because of line # 176 and 177, this is not required

Reply:
Removed.